torch/library.py

```diff
@@ -83,7 +86,7 @@ def impl(self, op_name, fn, dispatch_key=''):
 
         Args:
             op_name: operator name (along with the overload) or OpOverload object.
-            fn: function that's the operator implementation for the input dispatch key.
+            fn: function that's the operator implementation for the input dispatch key or :meth:`~Fallthrough`.
```

Comment:
:class: instead of :meth: ? meth sounds like it's for a method

Reply:
Since this API expect a function, I guess we should make our special fallthrough object a function?